src/pages/sign-transaction/sign-transaction.tsx

```diff
@@ -39,23 +40,27 @@ function SignTransactionBase(): JSX.Element | null {
   const [, setFeeRate] = useFeeRateState();
   const schema = useFeeSchema();
 
-  const onSubmit = useCallback(async () => {
-    setIsLoading();
-    await handleBroadcastTransaction();
-    setIsIdle();
-    setFeeEstimations([]);
-    setFee(null);
-    setFeeRate(null);
-    return () => setBroadcastError(null);
-  }, [
-    handleBroadcastTransaction,
-    setBroadcastError,
-    setFee,
-    setFeeEstimations,
-    setFeeRate,
-    setIsIdle,
-    setIsLoading,
-  ]);
+  const onSubmit = useCallback(
+    async values => {
+      setFee(stxToMicroStx(values.txFee).toNumber());
+      setIsLoading();
+      await handleBroadcastTransaction();
+      setIsIdle();
+      setFeeEstimations([]);
+      setFee(null);
+      setFeeRate(null);
```

Comment:
Are we using `setFeeRate` anywhere? Can it be deleted?

Reply:
That is a good question, now that we aren't using it to calculate the tx fee with the send max button I don't think it is needed. I'll double check and remove it if can.

Reply:
It's being used in this utils function, but I'll look at if we even need to calculate the fee here now ...maybe these can be removed now? I'll confirm.

https://github.com/hirosystems/stacks-wallet-web/blob/f086ec001f7aa58317de093248b068a7584e9edd/src/store/transactions/utils.ts#L18